python/paddle/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -61,4 +62,5 @@ if(WITH_DISTRIBUTE)
 endif()
 py_test_modules(test_parallel_executor_crf MODULES test_parallel_executor_crf SERIAL)
 py_test_modules(test_parallel_executor_fetch_feed MODULES test_parallel_executor_fetch_feed SERIAL)
+py_test_modules(test_dist_transformer MODULES test_dist_transformer SERIAL)
```

Comment:
We may add a timeout for this test here to avoid hanging up

Reply:
I have experienced hung issue for these 2 tests on CI. We can add that (or fix it) if it happens.